code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -76,6 +76,22 @@
 	required_reagents = list(/datum/reagent/consumable/corn_starch = 1, /datum/reagent/toxin/acid = 1)
 	required_temp = 374
 
+/datum/chemical_reaction/caramel
+	name = "Caramel"
+	id = /datum/reagent/consumable/caramel
```

Comment:
is this suppose to be a string?

Reply:
There isn't any consistency on reaction ids. Some are strings, some are the type. Apparently they just have to be unique and this is just that.

Reply:
lol ok

Reply:
easy gbp for whoever makes them either or